src/app.rs

```diff
@@ -1262,7 +1269,10 @@ impl App {
                         .collect::<Vec<SimplifiedPlaylist>>();
 
                     if !self.library.made_for_you_playlists.pages.is_empty() {
-                        self.library.made_for_you_playlists.pages[0]
+                        self.library
+                            .made_for_you_playlists
+                            .get_mut_results(None)
```

Comment:
This seems like a better way to handle getting a mutable reference to a page. What do you think?

Reply:
Looks great 👍